index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet"/>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"/> 
+
+    <title>Mohamed Aden Ighe: My personal portfolio</title>
+    
+<body>
+    <section id="part1">
+        <div class ="logo-placeholder">
+            <a href="#about" class="logo">MAI</a>
+        </div>
+        <div class="menu">
+            <nav class="material-symbols-outlined">Menu</nav>
+        </div>
+    </section>
```

Comment:
- [ ] Please let's implement the required changes from the previous reviewer. 🙏🏽🙏🏽
It is advisable to use html5 semantics. So how about you change this section to <code>header</code> instead?
For example:

```sh
<header id="part1">
_your code here_
</header>
```